website/docs/releases/2024/v2024.2.md

```diff
@@ -5,6 +5,8 @@ slug: /releases/2024.2
 
 :::::note
 2024.2 has not been released yet! We're publishing these release notes as a preview of what's to come, and for our awesome beta testers trying out release candidates.
+
```

Comment:
"To try the release candidate out" is, dunno, it sorta skips in my head.  Maybe "To try out the latest release candidate"?

Reply:
I agree yeah